tutorials-and-examples/hf-tgi/main.tf

```diff
@@ -146,6 +157,18 @@ resource "kubernetes_deployment" "inference_deployment" {
           empty_dir {}
         }
 
+        volume {
+          name = "gcs-fuse-csi-ephemeral"
```

Comment:
Need to run `terraform fmt main.tf` on this file to pass the first pre-submit check. It changes this line and line 166 as well.